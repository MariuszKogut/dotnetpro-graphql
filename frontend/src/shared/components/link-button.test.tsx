import React from 'react'
import { render } from '@testing-library/react'
import LinkButton from './link-button'

jest.mock('react-router-dom', () => ({
  useHistory: jest.fn(),
}))

describe('LinkButton', () => {
  it('renders without errors', () => {
    const { container } = render(
      <LinkButton to="http://www.handmade-systems.de" />,
    )
    expect(container).toMatchSnapshot()
  })
})
