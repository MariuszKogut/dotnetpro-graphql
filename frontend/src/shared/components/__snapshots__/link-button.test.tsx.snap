// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LinkButton renders without errors 1`] = `
<div>
  <button
    class="btn btn-primary"
    type="button"
  />
</div>
`;
